### PR TITLE
Fix visco plastic latex

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

**Comment by gassmoeller** [comment]
Looks good. Ready to merge when the tester has finished.

**Comment by gassmoeller** [comment]
@naliboff please take a look at the changes of this PR, and particularly check if the \\v should be an \\eta or \\epsilon.

File: source/material_model/visco_plastic.cc [hunk 1]
```diff
@@ -582,16 +582,16 @@ namespace aspect
...
+                                   "dislocation ($v_{disl}$, $n>1$, $m=0$) or composite "
                                    "$\\frac{v_{diff}*v_{disl}}{v_{diff}+v_{disl}}$ equation form. "
                                    "\n\n "
                                    "Plasticity limits viscous stress through a Drucker Prager "
                                    "yield criterion, where the yield stress in 3D is  "
                                    "$\\sigma_y = \\frac{6*C*\\cos(\\phi) + 2*P*\\sin(\\phi)} "
                                    "{\\sqrt(3)*(3+\\sin(\\phi))}$ "
                                    "and "
-                                   "$\\simga_y = C*\\cos(\\phi) + P*\\sin(\\phi)$ "
+                                   "$\\sigma_y = C*\\cos(\\phi) + P*\\sin(\\phi)$ "
```
Comment (bangerth) [inline source/material_model/visco_plastic.cc]: while you're here, mind to remove the asterisks?

File: source/material_model/visco_plastic.cc [hunk 2]
```diff
@@ -582,16 +582,16 @@ namespace aspect
...
                                    "\n\n "
-                                   "One may select to use the diffusion ($\\v_{diff}$; $n=1$, $m!=0$), "
-                                   "dislocation ($\\v_{disl}$, $n>1$, $m=0$) or composite "
+                                   "One may select to use the diffusion ($v_{diff}$; $n=1$, $m!=0$), "
+                                   "dislocation ($v_{disl}$, $n>1$, $m=0$) or composite "
                                    "$\\frac{v_{diff}*v_{disl}}{v_{diff}+v_{disl}}$ equation form. "
                                    "\n\n "
                                    "Plasticity limits viscous stress through a Drucker Prager "
                                    "yield criterion, where the yield stress in 3D is  "
                                    "$\\sigma_y = \\frac{6*C*\\cos(\\phi) + 2*P*\\sin(\\phi)} "
```
Comment (bangerth) [inline source/material_model/visco_plastic.cc]: same here, all of the asterisks are not necessary

**Comment by naliboff** [comment]
@tjhei - looks good, thanks for the clarity improvements.

**Comment by gassmoeller** [comment]
But still the occurences of 'v' does need to be replaced by '\\\eta', does it not @naliboff ?

**Comment by tjhei** [comment]
He confirmed it is correct as is.

**Pull request merged**